src/viur/core/bones/phone.py

```diff
@@ -0,0 +1,129 @@
+import re
+import typing as t
+import json
+
+from viur.core.bones.string import StringBone
+from viur.core.bones.base import ReadFromClientError, ReadFromClientErrorSeverity
+from viur.core import conf
+
+from pathlib import Path
+
+DEFAULT_REGEX = r"^\+?(\d{1,3})[-\s]?(\d{1,4})[-\s]?(\d{1,4})[-\s]?(\d{1,9})$"
+
+COUNTRYFILE = Path(__file__).parent / "../languages/country_information.json"
```

Comment:
Can you add this file to the [`package-data`](https://github.com/viur-framework/viur-core/blob/dd0431bc14c2a2b897f2f54de092cac45b0d5e71/pyproject.toml#L70)? Otherwise, I think it wouldn't be part of the wheel later, or am I wrong @phorward? 

See [Data Files Support](https://setuptools.pypa.io/en/latest/userguide/datafiles.html)

Reply:
@KadirBalku yes, please reformat the JSON to be pretty-printed, and add it to the package-data section.